cmd_image_build.go

```diff
@@ -72,9 +134,35 @@ func init() {
 			return nil
 		},
 	}
-	cmd.Flags().StringVar(&argBase, "base", "", "Use `IN_IMAGEFILE` as the base of the image")
-	cmd.Flags().StringVarP(&argTag, "tag", "t", "", "Tag the resulting image as `TAG`")
-	cmd.Flags().StringVar(&argEntrypoint, "entrypoint", "", "Set the resulting image's `ENTRYPOINT`")
+
+	cmd.Flags().StringVar(&flags.base, "base", "", "Use `IN_IMAGEFILE` as the base of the image")
+	cmd.Flags().StringVarP(&flags.tag, "tag", "t", "", "Tag the resulting image as `TAG`")
```

Comment:
OK, this is _horrible_. You have to write the function body and _then_ the flag parsing? Why in the world is it not considered better to use a named function _after_ the parsing??